### PR TITLE
Add Pun Pun Bangkok Bicycle Share, Thailand

### PR DESCRIPTION
**Pull request by BenSto**
This is a Provider from Thailand called Pun Pun Bikeshare. Unfortunately their Webpage is in thai, so Im not so sure about the real Company Name. To help myself I took their Tag from their Facebook Page, which is in English. Feel free to review. Thanks a lot! :bike:

### CODE REVIEW

File: pybikes/punpunbikeshare.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            bikes = bikes_counter
+            free = free_docks_counter
+            extra = {
+                    'slots' : item['bikeDockCount'],
+                    'address' : item['location']
+            }
+            station = PunpunbikeshareStation(name, latitude, longitude,
+                                             bikes, free, extra)
+            stations.append(station)
+            self.stations = stations
```
Comment (eskerda) [inline pybikes/punpunbikeshare.py]: This line should be outside of the loop.

**Comment by eskerda** [comment]
There's actually good information about bicycle ids. Let's take that too!

**Comment by eskerda** [comment]
It is a safe assumption it's operated by https://en.wikipedia.org/wiki/BTS_Group_Holdings

**Comment by eskerda** [comment]
Thanks for the initial commit! Merged your commit alongside my code changes, hope you don't mind.

**Comment by BenSto** [comment]
> Thanks for the initial commit! Merged your commit alongside my code changes, hope you don't mind.

No its fine, thank you very much!

**Pull request merged**